### PR TITLE
Update version

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: zap/src/main/java/org/zaproxy/zap/extension/pscan/scanner/ScriptsPassiveScanner.java [hunk 1]
```diff
@@ -121,15 +121,11 @@ private boolean appliesToCurrentHistoryType(ScriptWrapper wrapper, PassiveScript
         }
     }
 
-    /** @since TODO add version */
-    @Override
-    public AlertBuilder newAlert() {
-        return super.newAlert();
-    }
-
```
Comment (kingthorin) [inline zap/src/main/java/org/zaproxy/zap/extension/pscan/scanner/ScriptsPassiveScanner.java]: Was this removed intentionally?

Comment (kingthorin) [reply zap/src/main/java/org/zaproxy/zap/extension/pscan/scanner/ScriptsPassiveScanner.java]: Oops that shouldn’t have been an approve till this is discussed...

Comment (psiinon) [reply zap/src/main/java/org/zaproxy/zap/extension/pscan/scanner/ScriptsPassiveScanner.java]: Arg no, missed that :(

**Comment by psiinon** [comment]
Fixed and reverted the brackets.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**